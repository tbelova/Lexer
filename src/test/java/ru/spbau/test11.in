f(x) {
	x  = 10;
	return x * 5;
}
