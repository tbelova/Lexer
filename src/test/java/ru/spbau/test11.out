line 2:4
token recognition error at: '= '